view/ViewableData.php

```diff
@@ -358,44 +358,37 @@ public function renderWith($template, $customFields = null) {
 	 */
 	public function obj($fieldName, $arguments = null, $forceReturnedObject = true, $cache = false, $cacheName = null) {
 		if(!$cacheName) $cacheName = $arguments ? $fieldName . implode(',', $arguments) : $fieldName;
-		
+
 		if(!isset($this->objCache[$cacheName])) {
 			// HACK: Don't call the deprecated FormField::Name() method
```

Comment:
I think we can finally remove this, as FormField::Name is gone. now `$methodIsAllowed` is always true.